static/images/metal/fake-ui.svg

```diff
@@ -0,0 +1,32 @@
+<svg width="546" height="344" viewBox="0 0 546 344" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If the fake UI is dead code, remember to remove this and other fake-ui related resources.